dali/operators/geometry/affine_transforms/combine_transforms.cc

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <utility>
+#include <vector>
+#include "dali/core/format.h"
+#include "dali/core/geom/mat.h"
+#include "dali/core/static_switch.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/pipeline/data/types.h"
+#include "dali/pipeline/operator/op_spec.h"
+#include "dali/pipeline/workspace/workspace.h"
+#include "dali/pipeline/operator/operator.h"
+
+#define TRANSFORM_INPUT_TYPES (float)
+
+namespace dali {
+
+template <int... values>
+using dims = std::integer_sequence<int, values...>;
+
+template <typename T, int mat_dim>
+using affine_mat_t = mat<mat_dim, mat_dim, T>;
+
+DALI_SCHEMA(CombineTransforms)
```

Comment:
```suggestion
DALI_SCHEMA(TransformCombine)
```
I know that "combine transforms" is how you'd say it in English, we'll likely end up with "transforms.Combine" anyway